source/Buoy-Metaprogramming/Namespace.class.st

```diff
@@ -16,6 +16,14 @@ Namespace >> >> aName [
 	^ self at: aName
 ]
 
+{ #category : #'private - preconditions' }
+Namespace >> assertIsSymbol: aSymbol [
+
+	aSymbol isSymbol
+		ifFalse: [ AssertionFailed
+				signal: ( 'Only symbols can be bound. <1p> is not valid' expandMacrosWith: aSymbol ) ]
```

Comment:
```suggestion
	AssertionChecker 
		enforce: [ aSymbol isSymbol] 
		because: [ 'Only symbols can be bound. <1p> is not valid' expandMacrosWith: aSymbol ]
```